heudiconv/bids.py

```diff
@@ -10,8 +10,7 @@
 from random import sample
 from glob import glob
 
-import dicom as dcm
-import dcmstack as ds
+from heudiconv.external.pydicom import dcm
```

Comment:
use `from .external.pydicom ...`

Reply:
see my other comment... just an example: happen we decide to move `bids` into a submodule, would need to adjust all the relative imports.... here is a quick googled up echo of my opinion: https://softwareengineering.stackexchange.com/a/159505